backend/utils/feature-flags.go

```diff
@@ -76,3 +76,8 @@ func GetFeatureFlags(ctx *gin.Context) FeatureFlags {
 
 	return featureFlags
 }
+
+// ReportFlagActivation reports flag activation as a header
+func ReportFlagActivation(ctx *gin.Context, flag string, variant string) {
```

Comment:
Variants can theoretically be undefined in Unleash, right? Based on what type of feature flag

Reply:
Tanks, handling that now